lib/src/window_manager.dart

```diff
@@ -177,11 +190,41 @@ class WindowManager {
 
   /// Returns `bool` - Whether app is active
   ///
-  /// @platforms macos
+  /// @platforms macos,windows
   Future<bool> isActive() async {
     return await _channel.invokeMethod('isActive');
   }
 
+  /// Returns `bool` - Whether app is hidden on Blur Event
+  ///
+  /// @platforms windows
+  Future<bool> isHiddenOnBlur() async {
+    return await _channel.invokeMethod('isHiddenOnBlur');
+  }
+
+  /// Sets whether the window is hidden on Blur event.
+  Future<void> setHiddenOnBlur({bool hiddenOnBlur = false}) async {
```

Comment:
This method is only used when  IsHideOnDeactivate is true.
And differences are:
- as hide on deactivate method means we have to hide the window on the outside click
- and when we touch the outside of the window, the blur method is getting called 1st
- so we are hiding the window on the blur event and setting the HiddenOnBlur flag to true and after 500 milli-seconds we are resetting it to false 
- we are using set and reset logic as mentioned in the above point, it's because of :
	- when isHideOnDeactivate is on	:
		- whenever we are going to click the tray icon first blur method will get called the cause of the outside click and then the tray icon click will be called 
		- so on blur event :
			- it hides window if its already open
		- and on click event : 
			- it will again show the window as it's hidden by the blur method
		- and we have to reset that flag because :
			- when the user touch the outside window but not on tray icon, in this scenario we have to reset flag else we have to click two times on tray icon
	- when is off :
		things work as normally works